library/venv.po

```diff
@@ -441,14 +443,12 @@ msgstr ""
 "n'importe quel dossier existant cible, avant de créer l'environnement."
 
 #: ../Doc/library/venv.rst:116
-#, fuzzy
 msgid ""
 "``symlinks`` -- a Boolean value indicating whether to attempt to symlink the "
 "Python binary rather than copying."
 msgstr ""
 "``symlinks`` -- Une valeur booléenne qui indique si il faut créer un lien "
-"symbolique de la bibliothèque (et tous les DLLs ou autres binaires "
-"nécessaires, par exemple ``pythonw.exe``), plutôt que de la copier."
+"symbolique au lieu de copier."
```

Comment:
Il manque une information (*to the Python binary*)

Reply:
```suggestion
"symbolique sur le binaire Python au lieu de le copier."
```